src/clj/game/cards/ice.clj

```diff
@@ -2120,33 +2120,33 @@
 
    "Slot Machine"
    (letfn [(name-builder [card] (str (:title card) " (" (:type card) ")"))
-           (top-3 [runner] (take 3 (:deck runner)))
-           (top-3-names [runner] (map name-builder (top-3 runner)))
-           (top-3-types [runner] (->> (top-3 runner) (map :type) (into #{}) count))]
+           (top-3 [state] (take 3 (get-in @state [:runner :deck])))
```

Comment:
Alternatively you could use `(take 3 (rest (:deck runner)))`, but reloading the deck from state is probably best.

Reply:
I thought of that, but it felt cleaner to do what I actually want to do, which is get the *actual* current deck. Felt a little too magical to have to remember that the deck has changed since binding `runner`. It made me wish for a `get-runner` function, like in tests, lol